src/astro/sql/operators/load_table.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Callable, Optional
+
+from airflow.decorators.base import get_unique_task_id
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.databases import create_database
+
+
+class LoadTableOperator(BaseOperator):
+    """
+    Load the result of a SQL statement
+
+    :param sql_statement: sql query to execute
+    :param conn_id: Airflow connection id
+    :param handler: A Python callable handler to run on sqlalchemy legacy cursor result
+    """
+
+    template_fields = ("sql_statement", "conn_id")
+
+    def __init__(
+        self,
+        sql_statement: str,
+        conn_id: str,
+        handler: Optional[Callable] = None,
+        task_id: str = "",
+        **kwargs
+    ):
+        task_id = task_id or get_unique_task_id(
+            "get_value_list", dag=kwargs.get("dag"), task_group=kwargs.get("task_group")
+        )
+        self.sql_statement = sql_statement
+        self.conn_id = conn_id
+        self.handler = handler
+        super().__init__(task_id=task_id, **kwargs)
+
+    def execute(self, context: Context) -> Any:
+        db = create_database(self.conn_id)
+        result_set = db.run_sql(self.sql_statement)
```

Comment:
@pankajastro can you try this with apache-airflow-providers-snowflake=3.2.0? We faced a [`bug`](https://github.com/astronomer/astro-sdk/pull/686) yesterday. Just wanted to make sure this works with it.

Reply:
tested it worked.